.github/workflows/benchmark-wildcat.yml

```diff
@@ -0,0 +1,116 @@
+name: Benchmark GTSFM on select datasets on the wildcat machine
+
+################################################################################
+# Please only run this action after making sure that the shoter benchmarks
+# (defined in benchmark.yml) all run correctly and show no major regressions.
+#
+# Please be conscious of multiple queued actions, as they run sychronously on
+# one machine.
+################################################################################
+
+on: [pull_request, workflow_dispatch]
+jobs:
+  benchmark:
+    name: Benchmark
+    runs-on: self-hosted
+    strategy:
+      # Prevents other configs from being canceled if one of them fails.
+      fail-fast: false
+      matrix:
+        config_dataset_info: [
+            # config                     dataset    lookahead    img-extension  source  loader      max-res  share-intrinsics
+            [
+              deep_front_end,
+              skydio-501,
+              15,
+              jpg,
+              wget,
+              colmap-loader,
+              480,
+              true,
+            ],
+          ]
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    env:
+      PYTHON_VERSION: 3.8
+
+    steps:
+      - uses: actions/checkout@v3.5.2
+      - name: Cache frontend
+        uses: actions/cache@v3
+        env:
+          # Increase this value to reset cache
+          CACHE_NUMBER_FRONTEND: 0
+        with:
+          path: cache
+          key: ${{ matrix.config_dataset_info[0] }}-${{ matrix.config_dataset_info[1] }}-${{ matrix.config_dataset_info[2] }}-${{ matrix.config_dataset_info[6] }}-${{ env.CACHE_NUMBER_FRONTEND }}
+      ##########################################################################
+      # Notes (travisdriver):
```

Comment:
could you make this less verbose (and not in first person)?

Reply:
I could just add these comments to the PR and take them out of the bash file?

Reply:
Yes, could you replace lines 53-65 with `TODO: setup micromamba/miniconda to update env automatically.`

Reply:
done